cmake/Sandbox.conf.cmake

```diff
@@ -7,32 +7,32 @@
 [fdbmonitor]
 
 [general]
-restart_delay = 10
-## by default, restart_backoff = restart_delay_reset_interval = restart_delay
-# initial_restart_delay = 0
-# restart_backoff = 60
-# restart_delay_reset_interval = 60
-cluster_file = ${CMAKE_BINARY_DIR}/fdb.cluster
-# delete_envvars =
-# kill_on_configuration_change = true
+restart-delay = 10
```

Comment:
Well, seeing this made me realize that we have another source of underscores, which is in `fdbmonitor`. Some of the parameters here come from fdbserver (or similar binaries), but some of them are handled by `fdbmonitor` directly. I think we may need to update `fdbmonitor.cpp` and `FDBService.cpp`. Let me see if there are any other implications for that.

Reply:
Done. Thanks for pointing them out.